distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -2870,6 +2870,81 @@ public void testInjectAttributesOnUpdate() throws Exception {
         }
     }
 
+    @Test
+    public void testUploaderPolicyAllowsUploaderToQueryAndUpdateMetacard() throws Exception {
+        String xmlFileContent = getFileContent(XML_RECORD_RESOURCE_PATH + "/sampleCleanMetacard.xml");
+        String id = given().auth()
+                .preemptive()
+                .basic(ADMIN, ADMIN)
+                .body(xmlFileContent)
+                .header(HttpHeaders.CONTENT_TYPE, "text/xml")
+                .expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_CREATED)
+                .when()
+                .post(REST_PATH.getUrl())
+                .getHeader("id");
+        try {
+            String url = REST_PATH.getUrl() + id;
+
+            getServiceManager().startFeature(true, "sample-filter");
```

Comment:
❗️ We may want to add a `serviceManager.waitForRequiredBundles("sample-metacard-filter")` call here to make sure the bundle is really up and ready.